src/endpoints/rooms/createRoom.js

```diff
@@ -19,21 +19,17 @@ function handle(req, res) {
 	}
 	
 	var name = req.body.name; 
-	var owner_id = req.body.ownerId;
-	var val = Math.floor(1000 + Math.random() * 9000);

```

Comment:
Doesn't deleting this mean that rooms aren't created with codes?

Reply:
No I had already made a method to generate codes a while back so I just deleted this b/c I know mine works.